common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockHandler.java

```diff
@@ -176,7 +176,8 @@ private void checkAuth(TransportClient client, String appId) {
   /**
    * A simple class to wrap all shuffle service wrapper metrics
    */
-  private class ShuffleMetrics implements MetricSet {
+  @VisibleForTesting
```

Comment:
`@VisibleForTesting` causes classpath issues. Please note this in the java doc instead (SPARK-11615).

This is a scalastyle output, would be better to remove this annotation.